My Spotify/My Spotify/Network Layer/APIManager.swift

```diff
@@ -51,6 +51,27 @@ class APIManager {
         }
     }
     
+    //MARK: - Load image from URL api
```

Comment:
Where this method is used ?

Reply:
I didn't used but is came from develop from rebase